fix/mpassit/histlist_2d

```diff
@@ -9,12 +9,13 @@ lh				LH
 sfc_albedo                      ALBEDO
 hpbl                            PBLH
 ivgtyp                          IVGTYP
+isltyp                          ISLTYP
 snowfallac                      SNOWFALLAC
 vegfra                          VEGFRA
 shdmin                          SHDMIN
 shdmax                          SHDMAX
 canwat                          CANWAT
 snowc                           SNOWC
-snownc                          SNOWNC
-graupelnc                       GRAUPELNC
+snownc                          ACSNOW
+graupelnc                       ACGRAUP
```

Comment:
@EricJames-NOAA Could you double check these two line changes? Thanks!

Reply:
@chunhuazhou Check this PR:
https://github.com/NOAA-EMC/rrfs-workflow/pull/624/files

It will be good to check with @EricJames-NOAA whether changes to Lines 19-20 are needed. Thanks!

Reply:
Thanks, @EricJames-NOAA